mockmaker/bytebuddy/main/java/org/mockito/internal/creation/bytebuddy/MockBytecodeGenerator.java

```diff
@@ -61,18 +49,16 @@ public MockBytecodeGenerator() {
                              .intercept(to(MockMethodInterceptor.ForWriteReplace.class));
         }
         return builder.make()
-                      .load(SearchingClassLoader.combineLoadersOf(allMockedTypes(features)), ClassLoadingStrategy.Default.INJECTION)
+                      .load(new MultipleParentClassLoader.Builder()
+                              .append(features.mockedType)
+                              .append(features.interfaces)
+                              .append(MultipleParentClassLoader.class.getClassLoader())
+                              .append(Thread.currentThread().getContextClassLoader())
+                              .filter(isBootstrapClassLoader())
+                              .build(), ClassLoadingStrategy.Default.INJECTION)
```

Comment:
I'd like to split this one in another commit, same PR is ok though. Also `SearchingClassLoader` had issues with other tools like eclipse, maven surefire, or spring (fixed along the road). Do you have tested them ?